docs/inputs/sv_callers.md

```diff
@@ -0,0 +1,158 @@
+
+## Configuring Conversions
+
+Adding a conversion step to your MAVIS run is as simple as adding that section to the input JSON config.
+
+The general structure of this section is as follows
+
+```jsonc
+{
+    "convert": {
+        "<ALIAS>": {
+            "file_type": "<TOOL OUTPUT TYPE>",
+            "name": "<TOOL NAME>",  // optional field for supported tools
+            "inputs": [
+                "/path/to/tool/output/file"
+            ]
+        }
+    }
+}
+```
+
+A full version of the input configuration file specification can be found in the [configuration](../configuration/general.md) section.
+
+## Supported Tools
+
+The tools and versions currently supported are given below. Versions listed indicate the version of the tool for which output files have been tested as input into MAVIS. MAVIS also supports a [general VCF input](#general-vcf-inputs).
+
+| SV Caller                                                                   | Version(s) Tested | Files used as MAVIS input                     |
+| --------------------------------------------------------------------------- | ----------------- | --------------------------------------------- |
+| [BreakDancer (Chen, 2009)](../../background/citations#chen-2009)            | `1.4.5`           | `Tools main output file(s)`                   |
+| [BreakSeq (Abyzov, 2015)](../../background/citations#abyzov-2015)           | `2.2`             | `work/breakseq.vcf.gz`                        |
+| [Chimerascan (Iyer, 2011)](../../background/citations#iyer-2011)            | `0.4.5`           | `*.bedpe`                                     |
+| [CNVnator (Abyzov, 2011)](../../background/citations#abyzov-2011)           | `0.3.3`           | `Tools main output file(s)`                   |
+| [CuteSV (Jiang, 2020)](../../background/citations#jiang-2020)               | `1.0.10`          | `*.vcf`                                       |
+| [DeFuse (McPherson. 2011)](../../background/citations#mcpherson-2011)       | `0.6.2`           | `results/results.classify.tsv`                |
+| [DELLY (Rausch, 2012)](../../background/citations#rausch-2012)              | `0.6.1` `0.7.3`   | `combined.vcf` (converted from bcf)           |
+| [Manta (Chen, 2016)](../../background/citations#chen-2016)                  | `1.0.0`           | `{diploidSV,somaticSV}.vcf`                   |
+| [Pindel (Ye, 2009)](../../background/citations#ye-2009)                     | `0.2.5b9`         | `Tools main output file(s)`                   |
+| [Sniffles (Sedlazeck, 2018)](../../background/citations#sedlazeck-2018)     | `1.0.12b`         | `*.vcf`                                       |
+| [STAR-Fusion (Haas, 2017)](../../background/citations#haas-2017)            | `1.4.0`           | `star-fusion.fusion_predictions.abridged.tsv` |
+| [Straglr (Chiu, 2021)](../../background/citations#chiu-2021)                |                   |
```

Comment:
Is there a missing `|` at the end of this line? Also no version or file information?

Reply:
Yup, though oddly it doesn't seem to affect the render at all. As far as I can tell this tool isn't versioned (see github)

Reply:
I've asked Readman if there's a [stable release version](https://github.com/bcgsc/straglr/issues/8), we shouldn't be using a non-versioned iteration of this tool.

Reply:
oic, I think I didn't look at the releases in enough detail. I just saw it was called "final release before publication" and didn't realize the tag was semanticly versioned still (i've never seen someone do that before). I've pinged the user on #317 to check there

Reply:
As discussed, seems like there are newer commits in master not tagged so I think we will leave this version blank for the purposes of this PR